servlet-build-legacy/pom.xml

```diff
@@ -87,6 +87,148 @@
                             </execution>
                         </executions>
                     </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-resources-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>distribution-configuration</id>
+                                <goals>
+                                    <goal>copy-resources</goal>
+                                </goals>
+                                <phase>process-classes</phase>
+                                <configuration>
+                                    <overwrite>true</overwrite>
+                                    <outputDirectory>${basedir}/target/${server.output.dir.prefix}-servlet-${server.output.dir.version}</outputDirectory>
+                                    <resources>
+                                        <resource>
+                                            <!-- Pull the shared content from servlet-dist-legacy -->
+                                            <directory>../servlet-dist-legacy/src/distribution/resources</directory>
+                                            <filtering>true</filtering>
```

Comment:
I'm not sure I understand this. If shared content is required shouldn't it live in the this maven module?

Reply:
@jamezp I think I don't understand you very well. The resources that are being copied here from servlet-dist-legacy to servlet-build-legacy are necessary to make servlet-build match to servlet-dist. Those resources are for both modules and they should live in only one module. The strategy used here was the same strategy used for normal dist and build, see for example build-legacy/pom.xml

Reply:
That does make sense. I do think it should be reversed though. The files should live in `servlet-build-legacy` and the `servlet-dist-legacy` should copy the resources. In a sense the `*-dist*` is built off the `*-build*`.

Reply:
Does changing this for servlet-build/dist-legacy imply to you change it for normal dist/build as well? We are generating the normal build/dist in the same way. In my opinion, we should not change the legacy builds, but I would like to know if you see correct to have different strategies for both or what's your opinion about how we are generating the normal legacy builds. I understand legacy dist/builds are in the middle of the step to being removed completely.

Reply:
Well I guess the dist/build is the same. Just seems backwards to me, but I think it's okay if that's how we've been doing it. I'd say there's nothing to fix.